libs/gltfio/include/gltfio/Animator.h

```diff
@@ -26,35 +26,53 @@ namespace details { struct FFilamentAsset; }
 struct AnimatorImpl;
 
 /**
- * Animator can be used for two things: (1) updating matrices in Transform components
- * according to glTF animation definitions and (2) updating bone matrices in Renderable components
- * according to glTF skin definitions.
+ * \class Animator Animator.h gltfio/Animator.h
```

Comment:
This particular line might seem pointless at first, but it forces doxygen to show the correctly qualified path when it tells devs which `#include` to use for the class.